administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -213,7 +213,7 @@ const ApplicationCard = ({
               href={createCardQuery ? `./cards/add${createCardQuery}` : undefined}
               icon='id-number'
               intent='primary'>
-              Karte erstellen
+              {cardCreated ? 'Karte erneut erstellen' : 'Karte erstellen'}
```

Comment:
I think we should make it more noticeable for the user when a card has already been created for this application.
Maybe we can make the user confirm a dialog when clicking on this button in this case? And/Or add a [success callout](https://blueprintjs.com/docs/#core/components/callout) before the button toolbar saying that a card has been created for this application?

Reply:
hm sounds reasonable but was not considered by ui/ux @hauf-toni

Reply:
As the deadline is quite soon, I suggest we do this in a later step / separate issue.